addons/armaos/functions/fnc_shell_findLoginUser.sqf

```diff
@@ -0,0 +1,44 @@
+/**
+ * Tries to find the given username in the list of local users. If found switches mode to password.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ * 2: Username <STRING>
+ *
+ * Results:
+ * None
+ */
+
+params ["_computer", "_username"];
+
+private _terminal = _computer getVariable "AE3_terminal";
+
+private _users = _computer getVariable [ "AE3_Userlist", [["topsykrett", "test123"]] ];
```

Comment:
Why do we need a default user here?
Empty list would be sufficient.

Reply:
Only for debugging reasons. I need start my testmission very often. If I forgot to assign the user module, there is always my test user. But you are right. I will remove that on all occurrences.

Reply:
removed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/351ca2f7548ca9717ff26f606317f2c7e56c0763